proxygen_cli/cli/command_credentials.py

```diff
@@ -58,20 +60,27 @@ def set(custom_pairs, force):
     # Check if base credentials are set
     base_credentials_set = all(
         current_credentials.get(field) is not None
-        for field in ["client_id", "client_secret", "username", "password"]
+        for field in ["username", "password"]
     )
 
     if not base_credentials_set or force:
-        client_id = click.prompt("Enter client_id")
-        client_secret = click.prompt("Enter client_secret", default="", show_default=False)
         username = click.prompt("Enter username", default="", show_default=False)
         password = click.prompt("Enter password", default="", show_default=False)
 
-        current_credentials["client_id"] = client_id
-        current_credentials["client_secret"] = client_secret
         current_credentials["username"] = username
         current_credentials["password"] = password
 
+    # Check if client credentials are set
+    client_credentials_set = all(
+        current_credentials.get(field) is not None
+        for field in ["client_id", "client_secret"]
+    )
+    if not client_credentials_set or force:
```

Comment:
Does this still mean force will always ask for client id and secret?

Reply:
Yes Alex We need to remove force as we want to this happen only when Client Credenntials (Client_Id and Client_secret) are not set as part of package or need update.